src/bp/nlu-core/entities/microsoft-extractor/microsoft-extractor.test.ts

```diff
@@ -175,7 +175,7 @@ describe('Microsoft Extract Multiple', () => {
             source: 'one',
             entityId: 'system.number',
             extractor: 'system',
-            unit: 'number'
+            unit: ''
```

Comment:
See, this is what I was talking about.

These tests are too sensible

Reply:
I kind of like it, at least I knew I broke something in the unit system (which should be tested, I don't want a date to have a banana type). 

For the unit in the frontend I agree, we could either have it in the backend as "if you don't have a unit take the type, it's implemented like that" or "We have some redundancy between type and unit, fix it in the UI".

Reply:
You would like it less if you had more than 1000 unit tests covering your code. But idc, leave it like this.